docs/config.json

```diff
@@ -1318,6 +1318,15 @@
               ],
               "slug": "/database-access/guides/oracle-self-hosted/"
             },
+            {
+              "title": "Self-Hosted ClickHouse (Preview)",
```

Comment:
is this list sorted by title or we are putting previews at the bottom?

Reply:
I think that it suppose to be sorted. Fixed